controllers/subscriptions_test.go

```diff
@@ -117,6 +117,9 @@ func TestEnsureSubscription(t *testing.T) {
 						expectedSubscription.Spec.CatalogSource = odfSub.Spec.CatalogSource
 						expectedSubscription.Spec.CatalogSourceNamespace = odfSub.Spec.CatalogSourceNamespace
 						assert.Equal(t, expectedSubscription.Spec, actualSubscription.Spec)
+					} else if expectedSubscription.Spec.Package == IbmSubscriptionPackage {
```

Comment:
Did we miss adding the test for the IBM subscription or this is to do something specific with the v1.7.0

Reply:
This was due to the second commit.